docs/reference/data-sources/push.md

```diff
@@ -0,0 +1,39 @@
+# Push source
+
+## Description
+
+Push sources allow feature values to be pushed to the online store in real time. This allows fresh feature values to be made available to applications. Push sources supercede the 
+[FeatureStore.write_to_online_store](https://rtd.feast.dev/en/latest/index.html#feast.feature_store.FeatureStore.write_to_online_store).
+
+Push sources can be used by multiple feature views. When data is pushed to a push source, feast propagates the feature values to all the consuming feature views.
+
+## Example
+### Defining a push source
+
+```python
+from feast import PushSource, ValueType, BigQuerySource, FeatureView, Feature
+
+push_source = PushSource(
+    name="push_source",
+    schema={"user_id": ValueType.INT64, "life_time_value": ValueType.INT64},
+    batch_source=BigQuerySource(table="test.test"),
+)
+
+fv = FeatureView(
+    name="feature view",
+    entities=["location_id"],
+    features=[Feature(name="life_time_value", dtype=ValueType.INT64)],
```

Comment:
hm the entities here is `location_id` but the push source has a schema with `user_id`, those should be same?

Reply:
done